test/integration/test_build.py

```diff
@@ -348,3 +348,20 @@ def test_missing_runner(cli, runtime, ee_tag, data_dir, tmp_path):
         )
 
     assert "ERROR - Missing Ansible Runner installation" in einfo.value.stdout
+
+
+@pytest.mark.test_all_runtimes
+def test_bad_ansible_cfg(cli, runtime, ee_tag, data_dir, tmp_path):
+    """
+    Test that the check_galaxy script will cause build failure with
+    incorrect ansible.cfg
+    """
+    ee_def = data_dir / 'v3' / 'check_galaxy' / 'ee-bad-ansible-cfg.yml'
+
+    with pytest.raises(subprocess.CalledProcessError) as einfo:
+        cli(
+            f'ansible-builder build -c {tmp_path} -f {ee_def} -t {ee_tag} '
+            f'--container-runtime={runtime} -v3'
+        )
+
+    assert "ERROR - 'ansible-galaxy' command not functioning as expected" in einfo.value.stdout
```

Comment:
In addition to this `assert`, we should add another that checks for the actual error emitted from `ansible-galaxy` (visible in the output thanks to using `-v3`) to make sure it is the faulty `ansible.cfg` causing the error and not something else. In this case, looks like the error string we want begins with:

```
ERROR: Error reading config file (/etc/ansible/ansible.cfg): Source contains parsing errors:
```

Reply:
Thanks for getting the exact line I needed!